transformer_engine/jax/dot.py

```diff
@@ -49,7 +49,8 @@ def fp8_dot(fp8_gemm_pkg: FP8GemmPackage,
                        contracting_dims=contracting_dims,
                        sharding_type=sharding_type,
                        dp_axis_name="",
-                       tp_axis_name="")
+                       tp_axis_name="",
+                       fsdp_axis_name=None)
```

Comment:
Why not align other axis names, such as `fsdp_axis_name=""`?

Reply:
Done